daffodil-test/src/test/scala/org/apache/daffodil/section13/zoned/TestZoned.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.section13.zoned
+
+import org.apache.daffodil.tdml.Runner
+import org.junit.AfterClass
+import org.junit.Test
+
+object TestZoned {
+  val testDir = "/org/apache/daffodil/section13/zoned/"
+  lazy val runner = Runner(testDir, "zoned.tdml")
+
+  @AfterClass def shutdown(): Unit = {
+    runner.reset
+  }
+
+}
+
+class TestZoned {
+  import TestZoned._
+
+  @Test def testZonedPatternFail01(): Unit = { runner.runOneTest("zoned_textNumberPattern_fail01") }
+  @Test def testZonedPatternFail02(): Unit = { runner.runOneTest("zoned_textNumberPattern_fail02") }
+  @Test def testZonedPatternFail03(): Unit = { runner.runOneTest("zoned_textNumberPattern_fail03") }
+  @Test def testZonedPatternFail04(): Unit = { runner.runOneTest("zoned_textNumberPattern_fail04") }
+  @Test def testZonedFloatFail01(): Unit = { runner.runOneTest("zoned_float_fail01") }
+  @Test def testZonedDoubleFail01(): Unit = { runner.runOneTest("zoned_double_fail01") }
+  //TODO: @Test def testZonedEncodingFail01(): Unit = { runner.runOneTest("zoned_encoding_fail01") }
```

Comment:
I don't see this test in a TDML file anywhere, was this fixed?

Reply:
So this was the test I was talking about when I wanted to know if there was any sort of list of the encodings that we supported and if there was an easy way to determine which ones were ASCII derived.  All this test would do is have an expected failure (SDE) when using a non-ASCII derived encoding.

I believe we decided that since there wasn't a simple list of supported encodings it wasn't worth the effort to set up the SDE for this case.  This test case can just be removed.